crates/provider-blobstore-azure/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "wasmcloud-provider-blobstore-azure"
+version = "0.1.0"
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+[dependencies]
+anyhow = { workspace = true }
+async-nats = { workspace = true }
+async-trait = { workspace = true }
+tokio = { workspace = true }
+path-clean = { workspace = true }
+serde = { workspace = true }
+tokio-stream = { workspace = true, features = ["fs"] }
+tokio-util = { workspace = true, features = ["io"] }
+tracing = { workspace = true }
+wasmcloud-provider-sdk = { workspace = true, features = ["otel"] }
+wrpc-interface-blobstore = { workspace = true }
+wrpc-transport = { workspace = true }
+base64 = { workspace = true }
+bytes = { workspace = true }
+
+simple_env_load = "0.2.0"
+# NOTE: Unlike the other azure crates we import, this one has a default feature of enabling reqwest
+# with the native-tls feature. So we disable all default features just for good measure, but this
+# was the culprit for having an issue with building the provider
+azure_storage_blobs = { version = "0.20", features = [
+    "enable_reqwest_rustls",
+], default-features = false }
+azure_storage = { version = "0.20", features = [
+    "enable_reqwest_rustls",
+], default-features = false }
+azure_core = { version = "0.20", features = [
+    "enable_reqwest_rustls",
+], default-features = false }
+time = { version = "0.3" }
+futures = { version = "0.3" }
+serde_json = "1.0.114"
```

Comment:
Could all of these be workspace dependencies please?
We would not need the comment either, because we import everything with `default-features = false` in the workspace and select the required features on per-crate basis